libs/application/templates/financial-aid/src/fields/AboutForm/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
```

Comment:
About the naming of this file, then I am not a huge fan of naming all files `index.ts`, I would rather like to have it `AboutForm/AboutForm.tsx`. What do you think?

Reply:
I completely agree, update! 💯